apps/fm/test/unit/fm_test.js

```diff
@@ -70,6 +83,72 @@ suite('FM', function() {
       assert.equal($('frequency').textContent, 87.5);
     });
 
+    test('compare new set frequency with aria-valuenow', function() {
+      var frequency = 95.7;
+      var freq = frequencyDialer.setFrequency(frequency);
+      assert.equal(parseFloat(
+        $('dialer-container').getAttribute('aria-valuenow')), frequency);
+    });
+
+    test('screen reader swipe up on dialer should advance 0.1 Hz', function() {
```

Comment:
This and the test below can be taken out as a reusable function where you can parametrize keycode and expected frequency.